plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/ServletClasspathProvider.java

```diff
@@ -25,17 +25,18 @@ public ServletClasspathProvider() {
       return new IClasspathEntry[0];
     } else {
       return resolveClasspathContainer(runtime);
-    } 
+    }
   }
 
   @Override
   public IClasspathEntry[] resolveClasspathContainer(IRuntime runtime) {
-    java.nio.file.Path cloudSdkPath = new CloudSdkProvider(null).getCloudSdkLocation().toPath();
+    java.nio.file.Path cloudSdkPath =
+        new CloudSdkProvider().getCloudSdk().getJavaAppEngineSdkPath();
     if (cloudSdkPath == null) {
       return new IClasspathEntry[0];
     };
-    String servletJar = cloudSdkPath + "/platform/google_appengine/google/appengine/tools/java/lib/shared/servlet-api.jar";
-    String jspJar = cloudSdkPath + "/platform/google_appengine/google/appengine/tools/java/lib/shared/jsp-api.jar";
+    String servletJar = cloudSdkPath + "/shared/servlet-api.jar";
```

Comment:
why is /shared not included in the part returned by getJavaAppEngineSdkPath()?

also, we should use the resolve() method in Path instead of string concatenation here